src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -6,10 +6,16 @@
 import java.io.FileWriter;
 
 public class WorkWithFile {
+    public static final String COMMA = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final int FIRST_INDEX = 0;
+    public static final int SECOND_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
         readDataFromFile(fromFileName);
-        writeDataToFile(toFileName, totalStatCalculation(readDataFromFile(fromFileName)));
+        calculateResult(readDataFromFile(fromFileName));
```

Comment:
again - the result of calling this method is ignored

please look through these lines attentively

Reply:
ok , i will try it again ,but I don't  completely understand what you mean(